packages/main-ui/src/pages/components/wallet_signin_popup.rs

```diff
@@ -17,7 +20,21 @@ pub fn WalletSigninPopup(
     let display_logo = logo.clone();
     rsx! {
         div { id, class,
-            div { class: "justify-start text-white font-bold text-xl/24", "{tr.title}" }
+            div { class: "flex flex-row justify-start gap-12",
+                button {
+                    class: "cursor-pointer",
+                    onclick: move |_| {
+                        tracing::debug!("backward button clicked");
+                        popup.open(rsx! {
+                            SignupPopup { class: "w-[400px] mx-[5px]", lang }
```

Comment:
```suggestion
                            SignupPopup { class: "w-full max-w-400 mx-5", lang }
```
어차피 이렇게 다 줄거면 signup popup 안에서 크기 조정을 하는게 더 나을거 같긴한에ㅛ.